src/graphics/program-lib/chunks/reproject.frag

```diff
@@ -167,6 +167,12 @@ mat3 matrixFromVector(vec3 n) { // frisvad
     return mat3(b1, b2, n);
 }
```

Comment:
Is this now unused? Any reason not to delete it and make `matrixFromVectorSlow` into `matrixFromVector`?

Reply:
I would like to keep it around for more accurate performance testing, but ultimately if we don't expose both as user options it can be removed.